### DISCUSSION TITLE
Paperless ngx update

### DISCUSSION THREAD

**Discussion by tomdh76** [tteck/Proxmox#559] [Help Needed] [answered]
Hi, I used the script update bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/paperless-ngx-update.sh)"
But after that I had to use python3 manage.py migrate (see https://github.com/paperless-ngx/paperless-ngx/issues/1651) , maybe this could automatically be integrated in the script?

**Comment by tteck** [comment]
It's already in the update script.

**Comment by tteck** [comment, marked as answer]
Changed migrate to long path.

**Reply by tomdh76** [reply]
Thx, I had now no problem updating from 1.9 to 1.92
